templates/registration/confirmation_email.html.twig

```diff
@@ -0,0 +1,10 @@
+<p>Hi,</p>
+
+<p>
+    Please confirm your email address by clicking the following link: <br><br>
+    <a href="{{ signedUrl|raw }}">Confirm my Email</a>.<br><br>
```

Comment:
why `|raw` ? This is wrong. HTML escaping must be applied

Reply:
Yeah interesting, this was in the generated template from makerbundle AFAIK, will check/report there if that's the case.